src/routes/search/components/SearchHeader.js

```diff
@@ -38,7 +38,7 @@ class SearchHeader extends React.Component<PropsType> {
         <BoxShadow theme={theme}>
           <HorizontalLeft theme={theme}>
             <HeaderBackButton theme={theme} onPress={closeSearchBar} />
-            <ThemedSearchBar theme={theme} onChangeText={onSearchChanged} value={query} />
+            <ThemedSearchBar theme={theme} onChangeText={onSearchChanged} value={query} autofocus />
```

Comment:
Regarding this discussion: https://github.com/Integreat/integreat-react-native-app/pull/237#issuecomment-617290530

Does this autofocus work when opening the search for the first time?

Reply:
Yes, works for me.